gulpfile.js

```diff
@@ -0,0 +1,127 @@
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var webpack = require('webpack');
+var WebpackDevServer = require('webpack-dev-server');
+var fs = require('fs');
+var rename = require('gulp-rename');
+var eslint = require('gulp-eslint');
+
+var gulpWebpack = require('gulp-webpack');
+
+var preprocess = require('gulp-preprocess');
+var header = require('gulp-header');
+
```

Comment:
nitpick: you don't need extra line returns here and the next few lines.

Reply:
ok